### DISCUSSION TITLE
1.3.10

### DISCUSSION THREAD

**Discussion by chucker** [chucker/Mastonaut#5] [Announcements]
This version is virtually the same as [the initial open-source commit](https://github.com/brunophilipe/Mastonaut/commit/d856953b1afd66e3bf31fb8264971e3c00aaa05e), and should be little different from the App Store 1.3.9 release.

This fork will not be published through the App Store. For software update purposes, it instead includes [Sparkle](https://sparkle-project.org/).

<hr /><em>This discussion was created from the release <a href='https://github.com/chucker/Mastonaut/releases/tag/app-1.3.10'>1.3.10</a>.</em>

**Comment by splorp** [comment]
> “… should be little different from the App Store 1.3.9 release.”

One different is that this release (and the 1.3.10 draft release) both require macOS 10.15 Catalina, whereas the 1.3.9 App Store release only requires macOS 10.13 High Sierra.
